LemonUI/Scaleform/BaseScaleform.cs

```diff
@@ -56,19 +56,32 @@ public abstract class BaseScaleform : IScaleform
         public BaseScaleform(string sc)
         {
             Name = sc;
+            
+            LoadScaleform();
+        }
+
+        #endregion
+
+        #region Protected Functions
+
+        /// <summary>
+        /// Load a given Scaleform
+        /// </summary>
+        protected void LoadScaleform()
```

Comment:
Ill recommend sending this changes in a separate PR.

Reply:
I'll make a new PR with only those changes